src/Plugin/QueueWorker/AbstractPostProcessorQueueWorker.php

```diff
@@ -347,15 +346,16 @@ public function processItem($data) {
         if (($inindex === 0 || $inkeystore === FALSE) ||
           $data->force == TRUE) {
           // Extract file and save it in key_value collection.
-          $this->logger->log(LogLevel::INFO, 'Invoking @plugin on ADO Node ID @nodeid.',
+          $this->logger->log(LogLevel::INFO, 'Invoking @plugin on ADO Node ID @nodeid@sequence.',
             [
               '@plugin' => $processor_instance->getPluginId(),
               '@nodeid' => $data->nid,
+              '@sequence' => !empty($data->sequence_number) ? ", page " . $data->sequence_number : "",
```

Comment:
Not always a page friend. This might be also a set of URLs from a Wacz file or any other ordering, let's put "sequence" here